esbuild.mjs

```diff
@@ -0,0 +1,16 @@
+import esbuild from "esbuild";
+import { NodeGlobalsPolyfillPlugin } from "@esbuild-plugins/node-globals-polyfill";
+import { NodeModulesPolyfillPlugin } from "@esbuild-plugins/node-modules-polyfill";
+
+esbuild.build({
+	entryPoints: [ "./VerifiableCredential/index.mjs" ],
+	outfile: "./dist/VerifiableCredential.mjs",
+	platform: "browser",
+	format: "esm",
+	bundle: true,
+	minify: true,
+	plugins: [
+		NodeGlobalsPolyfillPlugin(),
+		NodeModulesPolyfillPlugin(),
```

Comment:
are the polyfills needed to support `ajv`?

Reply:
`esbuild > 0.16.7` isn't compatible with `"@esbuild-plugins/node-globals-polyfill"` and `"@esbuild-plugins/node-modules-polyfill"` 😞 . [node-stdlib-browser](https://github.com/niksy/node-stdlib-browser) i think can replace both and allegedly seems to work much better.

Reply:
[relevant github issue](https://github.com/remorses/esbuild-plugins/issues/1)

Reply:
yeah that's why i went with `0.16.7` directly since that was the last version i know worked correctly.  i'd rather make updating this dependency a separate commit (and only if it's really actually needed)